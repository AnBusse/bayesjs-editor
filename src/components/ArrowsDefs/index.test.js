import { shallow } from 'enzyme';
import React from 'react';
import ArrowsDefs from './index';

const shallowComponent = () =>
  shallow(<ArrowsDefs />);

describe('ArrowsDefs Component', () => {
  it('matches snapshot', () => {
    const component = shallowComponent();

    expect(component).toMatchSnapshot();
  });
});
