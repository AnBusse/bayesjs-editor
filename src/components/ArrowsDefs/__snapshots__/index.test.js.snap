// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ArrowsDefs Component matches snapshot 1`] = `
<defs>
  <marker
    id="triangle"
    markerHeight="6"
    markerWidth="6"
    orient="auto"
    refX="8"
    refY="5"
    viewBox="0 0 10 10"
  >
    <path
      d="M0,0 L10,5 L0,10"
      fill="#333"
    />
  </marker>
  <marker
    id="#triangle-with-low-opacity"
    markerHeight="6"
    markerWidth="6"
    opacity="0.2"
    orient="auto"
    refX="8"
    refY="5"
    viewBox="0 0 10 10"
  >
    <path
      d="M0,0 L10,5 L0,10"
      fill="#333"
    />
  </marker>
</defs>
`;
